plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotQueryBuilder.java

```diff
@@ -81,7 +84,11 @@ public static String generatePql(PinotTableHandle tableHandle, List<PinotColumnH
         }
         pqlBuilder.append(" LIMIT ")
                 .append(limit);
-        return pqlBuilder.toString();
+        String pql = pqlBuilder.toString();
+        if (log.isDebugEnabled()) {
+            log.debug("Generated PQL: %s", pql);
```

Comment:
Note that we already have the following line:
https://github.com/trinodb/trino/blob/58ff0c0f919b4fd2c00656fb18de4e749c070c30/plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotClient.java#L552

Reply:
Are you saying this is not needed anymore?